src/gentropy/method/l2g/trainer.py

```diff
@@ -105,7 +105,7 @@ def _get_shap_explanation(
                 data=training_data,
                 feature_perturbation="interventional",
             )
-            return explainer(training_data)
+            return explainer(training_data, check_additivity=False)
```

Comment:
This should be optional. The check should be true by default

Reply:
I will check the error and produce the result without the additivity check if it is raised.